tests/js/client/shell/shell-foxx-interface.js

```diff
@@ -168,8 +170,9 @@ function foxxInterfaceSuite () {
     testFoxxInterfaceHeadBinary: function () {
       let res = arango.HEAD_RAW(binUrl);
       assertEqual(res.code, 200, res.parsedBody);
-      assertTrue(res.body instanceof Buffer);
-      assertEqual(res.body.length, 0);
+      // we don't create bodies for head requests anymore:
+      //assertTrue(res.body instanceof Buffer);
+      //assertEqual(res.body.length, 0);
```

Comment:
This should probably be `assertUndefined(res.body);` like in the other tests then.